superset/viz.py

```diff
@@ -327,7 +327,7 @@ def get_json(self):
             self.get_payload(),
             default=utils.json_int_dttm_ser, ignore_nan=True)
 
-    def cache_key(self, query_obj):
+    def cache_key(self, query_obj, **extra):
```

Comment:
It'd be nice to add something in the docstring about what extra is used for currently, and could be used for in the future.